aerogearcatalog/sync-app-apb
# AEROGEAR-8509 - move data sync dashboard form old apb and make it a t…

…emplate

Copied from https://github.com/aerogearcatalog/data-sync-apb/blob/master/roles/provision-data-sync-apb/files/data-sync-dashboard.json and replaced the hardcoded app names with `{{ SYNC_APP_NAME }}`

Provisioning using this template is part of another task.


Verification:
* Provovision a sample app with sync-app-apb. Make sure that app has a metrics endpoint.
* Add `org.aerogear.metrics/plain_endpoint: "/metrics"` to the **pod** of the app
* Try this PR file in Grafana. Replace the {{ SYNC_APP_APB }} with the sync app you chose during sync-app-apb provisioning.

I set it up here already: 
* https://grafana-ggg.apps.aliok-64ea.openshiftworkshop.com/dashboard/db/data-sync-test?orgId=1&from=now-5m&to=now
* https://sync-app-test-ggg.apps.aliok-64ea.openshiftworkshop.com/graphql
